spec/models/category_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :name }
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :name } 
+  end
+
```

Comment:
You need to test the association between the category and dishes.
This needs to be done in both model specs

https://github.com/thoughtbot/shoulda-matchers

Reply:
Done